benchmarks/src/main/scala/zio/QueueChunkBenchmark.scala

```diff
@@ -0,0 +1,66 @@
+package zio
+
+import org.openjdk.jmh.annotations._
+import zio.IOBenchmarks._
+
+import java.util.concurrent.TimeUnit
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Measurement(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 3)
+@Warmup(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 3)
+@Fork(3)
+/**
+ * This benchmark offers and takes chunks of items.
+ */
+class QueueChunkBenchmark {
+
+  val totalSize   = 1000
+  val chunkSize   = 10
+  val parallelism = 5
+
+  val chunk: List[Int] = List.fill(chunkSize)(0)
+
+  var zioQ: Queue[Int] = _
+
+  @Setup(Level.Trial)
+  def createQueues(): Unit =
+    zioQ = unsafeRun(Queue.bounded[Int](totalSize))
+
+  @Benchmark
+  def zioQueueParallel(): Int = {
+
+    val io = for {
+      offers <- IO.forkAll {
```

Comment:
`forkAll_` since you don't want to measure the overhead of the join concatenation.

Reply:
I don't think we can use `forkAll` here because we need a fiber that we can await to ensure that the work was completed.